app/sidekiq/benefits_intake_status_job.rb

```diff
@@ -130,4 +117,10 @@ def log_result(result, form_id, uuid, time_to_transition = nil, error_message =
       Rails.logger.info('BenefitsIntakeStatusJob', result:, form_id:, uuid:, time_to_transition:)
     end
   end
+
+  def form_submission_attempts_hash
+    @_form_submission_attempts_hash ||= FormSubmissionAttempt
```

Comment:
Interesting syntax, was the `@_` a typo?

Reply:
It's not a typo. I think it's a naming convention when you have a variable that you don't anticipate using in other contexts. So `@_form_submission_attempts_hash` is never directly used, we just refer to `form_submission_attempts_hash`, the memoized method. That's my understanding anyway. 🙂